### PR TITLE
Fixes #1 Migration of content from initial spec release

### LINKED ISSUE #1: Migration of content from initial spec release
As we move from the whole specification being stored in a single repo to one distributed across many repos, we need to copy content across.

This issue will form the basis for that copy (and paste!)

### PR DESCRIPTION
**Pull request by rachellawson**
Adding the files from the original spec "as is" though I think a follow up to make better image filnames might be nice.

### CODE REVIEW

File: spec/SUMMARY.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-* [9 Workflows](9-workflows.md)
-* [10 Other Resources](10-other-resources.md)
-* [11 Key Decision Log](11-key-decision-log.md)
-* [12 Future Considerations](12-future-considerations.md)
+* [7 Requirements Clarification ](7-requirements-clarification.md)
+* [8 Performance Requirements](8-performance-requirements.md)
+* [9 Data Structures](9-data-structures.md)
+* [10 Service APIs](10-service-apis.md)
+* [11 Workflows](11-workflows.md)
+* [13 Future Considerations](13-future-considerations.md)
```
Comment (rachellawson) [inline spec/SUMMARY.md:16]: I don't really understand why there isn't a 12 here - there wasn't in the original?

**Comment by rachellawson** [comment]
So far as reviewing, I would suggest checking that
- [ ] nothing other than in the spec folder is changed
- [ ] images are linked in markdown with correct paths
- [ ] there are no missing images
- [ ]

File: spec/SUMMARY.md (hunk 1 above)
Comment (taylordowns2000) [reply spec/SUMMARY.md:16]: @rachellawson , i think that section 12 is "Key Decision Log". Is that standard? I see it in the current IM spec.

**Comment by taylordowns2000** [comment]
@rachellawson , Amazing! Thank you! ❤️ 

Re: missing images, broken paths and review, i'd want a `npm run build` command to build and test the specification locally as it will look on the web. What's the equivalent for gitbook? (That `npm run build` command is for docusuarus. Docusaurus also validates all those paths and ensures that nothing's broken. There may be a similar feature in gitbook.)

File: spec/SUMMARY.md (hunk 1 above)
Comment (rachellawson) [reply spec/SUMMARY.md:16]: hmm - it seems it was missing from the migration from doc to original markdown. Now added in at this stage!

**Comment by rachellawson** [comment]
GitBook will auto-build a preview and attach a link to the PR once it is connected. BUT I need the initial content in there to configure that and this import contains that content. So we will see previews on _subsequent_ PRs…

**Review by mcarlson — APPROVED** [review]

**Comment by taylordowns2000** [comment]
Amazing! Thank you!

Re: missing images, broken paths and review, i'd want a `npm run build`
command to build and test the specification locally as it will look on the
web. What's the equivalent for gitbook? (That `npm run build` command is
for docusuarus. Docusaurus also validates all those paths and ensures that
nothing's broken. There may be a similar feature in gitbook.)

**Comment by rachellawson** [comment]
So, that is probably a wider conversation, @taylordowns2000, and maybe needs an issue of its own but PRs should now automatically be prompting GitBook to build a preview and linking to that preview in the PR.

**Pull request merged**